build-tools/scripts/gettextdomains

```diff
@@ -32,13 +32,13 @@ function get_domains_and_err()
     fi
 
     for F in $SRCFILES; do
-        # strip comments from the files and match [_]_( "..." )
+        # strip comments from the files and match [_]_( "..." ), handle Ruby string interpolation
         # 1. perl: strip one-line-comments
```

Comment:
I think this comment is slightly better:
\# 1. perl: strip one-line-comments, except ruby "string #{interpolation}"